proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -56,6 +58,15 @@ message CreateNewAddressResponse {
   string address = 1;
 }
 
+message CreateSidechainProposalRequest {
+  google.protobuf.UInt32Value sidechain_id = 1;
+  SidechainDeclaration declaration = 2;
+}
+
+message CreateSidechainProposalResponse {
+  ReverseHex txid = 1;
```

Comment:
This is not going to work. Or, this assumes that we're running on regtest? The way I'm thinking about this: In this RPC we create the `OP_RETURN` output, and persist it to some sort of DB. Later on, the `OP_RETURN` gets included in a block. That could be through the block mining procedure in this repo (for regtest), or it could be through some other mechanism (signet++). 

Two ideas for what this RPC could return: 
1. Some sort of DB ID that can be used to read back the proposal
2. The created OP_RETURN message